packages/html/stories/DynamicLoading.stories.js

```diff
@@ -100,12 +99,13 @@ const Template = ({ label, ...args }) => {
       // Adds cells to the model in a single step
       graph.batchUpdate(() => {
         const xml = server(cell.id);
-        const doc = xmlUtils.parseXml(xml);
+        const doc = xmlUtils.parseXml(xml);        
```

Comment:
extra trailing spaces
```suggestion
        const doc = xmlUtils.parseXml(xml);
```